unittests/fixtures/factory/web-area/monitor/entry_el8_osg34

```diff
@@ -0,0 +1 @@
+entry_el8_osg35
```

Comment:
Should this be `..._osg34` instead because I see the file is named `entry_el8_osg34`?

Reply:
see above